### ISSUE TITLE
Capacitor support

### ISSUE THREAD

**Issue by kalepail** [karandpr/cordova-plugin-android-safetynet#4]
What are the chances you could port this over to a Capacitor plugin? I'm trying to get this to work in my project but it's not going great. Getting the pro to get it ported to Capacitor would be amazing 😊🥺

**Comment by karandpr** [comment]
Sure thing. 
I think I had a WIP plugin so I just have to find the motivation to complete it lol. 
I am a bit out of loop with Ionic, so it will probably take a month to something.

**Comment by kalepail** [comment]
That would be amazing ❤️

**Comment by karandpr** [comment]
@tyvdh ,. This is still in pipeline. There are some changes in SafetyNet that I want to cover. I will look into it this weekend.

**Comment by sneko** [comment]
Hi @karandpr , since the last public commit, any commit you have locally? Thank you,

EDIT: would need to give a try to https://www.npmjs.com/package/capacitor-firebase-appcheck

**Comment by karandpr** [comment]
@sneko 
Not really. I was working on a capacitor version and sadly the API docs for capacitor are strictly OK. (nowhere near Cordova)
After trudging through some plugin implementations, I was not really interested in continuing further.

Anyway, SafetyNet is deprecated & replaced by Play Integrity API. 
I can look into but not sure where the capacitor is heading, or how many people are even interested in such plugin.
